spec/ajv.spec.js

```diff
@@ -377,26 +377,70 @@ describe('Ajv', function () {
 
 
   describe('addFormat method', function() {
-    it('should add format as regular expression', function() {
-      ajv.addFormat('identifier', /^[a-z_$][a-z0-9_$]*$/i);
-      testFormat();
-    });
+    describe.skip('without any type restrictions specified', function() {
```

Comment:
If format is not explicitly declared in the schema then custom formatters should not be applied.